documentation/long-stay-baseline.mdx

```diff
@@ -0,0 +1,528 @@
+---|---
+LENGTH_OF_STAY             |1.000000
+AGE_ON_ADMISSION           |0.308769
+ae_arrival_mode_Ambulance  |0.213357
+IS_frailty_proxy           |0.205508
+IS_dementia                |0.158076
+IS_chronic_kidney_disease  |0.157948
+ae_arrival_mode_Friend / Relative Drop Off     |-0.150921
+IS_care_home_on_admission  |0.125849
+stroke_ward_stay           |0.121665
+MAIN_SPECIALTY_CODE_AT_ADMISSION_DESCRIPTION_Paediatrics      |-0.110607
+IS_diabetes                |0.103683
+ae_arrival_mode_Car       |-0.102031
+IS_cancer                  |0.088739
+MAIN_SPECIALTY_CODE_AT_ADMISSION_DESCRIPTION_Trauma and Orthopaedics       |0.085817
+MAIN_SPECIALTY_CODE_AT_ADMISSION_DESCRIPTION_Rehabilitation                  |0.085401
+IS_mental_health           |0.083227
+IS_illness_not_injury     |-0.075949
+MAIN_SPECIALTY_CODE_AT_ADMISSION_DESCRIPTION_General surgery          |-0.075311
+MAIN_SPECIALTY_CODE_AT_ADMISSION_DESCRIPTION_ENT                     | -0.073087
+IS_COPD                    |0.069720
+
+These indicate that age and age-related illness, as well as arrival mode are strong factors in determining length of stay.
+
+#### Variation inflation factors
+
+Variation inflation factors (VIF) confirmed the presence of multi-colinearity between a number of features (VIF > 10).
+
+#### Homoescadisticity
+
+A basic ordinary least squares (OLS) regression model was fitted to the full feature set, then residuals calculated. 
+
+Residuals failed Shapiro-Wilk, Kolmogorov-Smirnov and Anderson-Darling tests for normality, as well as visual inspection:
+
+![Plot of residuals for OLS model of length of stay](images/long-stay-baseline/residuals.jpeg)
+
+OLS methods were therefore excluded from modelling.
+
+## 6. Modelling<a name='modelling'></a>
+
+The machine learning modelling approach was as follows:
+
+1. Split the data into a training (70%), validation (15%) and test (15%) data set
+2. Check the data splits do not introduce selection bias for length of stay, age, sex, or ethnicity
+3. Train baseline models with default parameters on the training set
+4. Evaluate baseline models on the validation test
+5. Select the best performing model
+6. Tune the best performing model using cross-validation on the training and validation set
+7. Report the final performance of the model using the test set
+
+![Summary of machine learning approach](images/long-stay-baseline/ml-approach.png)
+
+Training, validation and test splits were representative of the population and did not introduce selection bias:
+
+**Length of stay**
+
+![Distribution of length of stay by data split](images/long-stay-baseline/split-los.jpeg)
+
+**Age**
+
+![Distribution of age by data split](images/long-stay-baseline/split-age.jpeg)
+
+**Sex**
+
+Proportion of `male`, `female` patients in each split:
+
+```
+train: [0.53, 0.47]
+validate: [0.51, 0.49]
+test: [0.53, 0.47]
+```
+
+**Ethnicity**
+
+Proportions of each ethnicity for each split:
+
+```
```

Comment:
There're a lot of `0.0` in this proportions. Maybe can clean it a bit by just showing the first ~7 largest group?